<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw</class>
 <widget class="QMainWindow" name="mw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mw</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose the mode of capturing </string>
    </property>
    <widget class="QRadioButton" name="rb_auto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto (Windows -&gt; Linux)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_semi">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Semi-auto (choose the parametres)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_manual">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual (intercept -&gt; manual modify -&gt; send)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_capture">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Captuing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_exit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbox_param">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose the parametres</string>
    </property>
    <widget class="QCheckBox" name="cb_ttl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TTL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_ip_src">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP source</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ttl_old">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Old TTL:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ttl_new">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>New TTL:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ttl_old">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ttl_new">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ip_old">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>96</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Old src:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_old_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_old_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_old_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_old_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dot_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_new_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_new_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_new_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ip_new">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>New src:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_src_new_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dot_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_old_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dot_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_new_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_new_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ip_old_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>186</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Old dst:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ip_new_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>New dst:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_old_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_new_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dot_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dot_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_ip_dst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP destination</string>
     </property>
    </widget>
    <widget class="QLabel" name="dot_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_old_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_new_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_ip_dst_old_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dot_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose the in/out adapters</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>OUT adapter:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IN adapter:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comb_adapter_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comb_adapter_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Tips: 0.0.0.0 old src/dst mean changing to &quot;new ip&quot; from any avaible ip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
